package.json

```diff
@@ -10,6 +10,7 @@
     "checkHTML": "site-validator --verbose --url",
     "checkHTMLLocal": "site-validator --verbose --local --url http://localhost:8080",
     "dev": "nodemon index.js",
+    "lint": "eslint index.js config.js knexfile.js urls.js lib/** controllers/**",
```

Comment:
c'est pas possible d'utiliser un "*", au lieu de faire "index.js config.js knexfile.js urls.js" ? Parce que si on rajoute un fichier il faudra penser à le rajouter là, on y pensera jamais...

Reply:
Malheureusement, avec l'organisation actuelle du code, ça plante si l'on met qq chose du genre `eslint *.js */**/*.js`.

```
Oops! Something went wrong! :(
ESLint couldn't find the config "@ljharb" to extend from. Please check that the name of the config is correct.

The config "@ljharb" was referenced from the config file in "/Users/pom/workspace/conferences/node_modules/array-includes/.eslintrc".
```
=> Il semble analyser dans node_modules un package qui a un fichier .eslintrc (qui plus est mal formé).

Or il ne devrait pas analyser quoi que ce soit dans node_modules (https://eslint.org/docs/user-guide/migrating-to-7.0.0#default-ignore-patterns-have-changed). 

Une option serait de mettre le code dans un répertoire src/ et de faire "eslint src/*.js src/**/*.js". De cette façon, on éviterait je pense de parser node_modules.

Reply:
There was a solution ;). https://github.com/eslint/eslint/issues/13896#issuecomment-736727587
Cool! En entourant de double quotes, ça fait ce qu'on veut :)